docs/design/usage-based-scheduling.md

```diff
@@ -56,16 +56,17 @@ tiers:
       - name: proportion
       - name: nodeorder
       - name: binpack
-metrics:                             # metrics server related configuration
-  type: prometheus                   # Optional, The metrics source type, prometheus by default, support prometheus and elasticsearch
-  address: http://192.168.0.10:9090  # mandatory, The metrics source address
-  interval: 30s                      # Optional, The scheduler pull metrics from Prometheus with this interval, 5s by default
-  tls:                               # Optional, The tls configuration
-    insecureSkipVerify: "false"      # Optional, Skip the certificate verification, false by default
-  elasticsearch:                     # Optional, The elasticsearch configuration
-    index: "custom-index-name"       # Optional, The elasticsearch index name, "metricbeat-*" by default
-    username: ""                     # Optional, The elasticsearch username
-    password: ""                     # Optional, The elasticsearch password
+metrics:                               # metrics server related configuration
+  type: prometheus                     # Optional, The metrics source type, prometheus by default, support prometheus and elasticsearch
+  address: http://192.168.0.10:9090    # Mandatory, The metrics source address
+  interval: 30s                        # Optional, The scheduler pull metrics from Prometheus with this interval, 5s by default
+  tls:                                 # Optional, The tls configuration
+    insecureSkipVerify: "false"        # Optional, Skip the certificate verification, false by default
+  elasticsearch:                       # Optional, The elasticsearch configuration
+    index: "custom-index-name"         # Optional, The elasticsearch index name, "metricbeat-*" by default
+    username: ""                       # Optional, The elasticsearch username
+    password: ""                       # Optional, The elasticsearch password
+    hostnameFieldName: "host.hostname" # Optional, The elasticsearch hostname field name, "host.hostname" by default

```

Comment:
Please forgive my limited understanding of elasticsearch, why do we need to support the configuration of `hostnameFieldName` here?

In addition, if you need to support dynamic configuration of es-related configuration, it is not appropriate to put it in the config file of the volcano scheduler. Is it better to create an independent es configuration file?

Reply:
@wangyang0616 That's not a dynamic configuration.  People may configure different hostname field other than `host.hostname`